pkg/volume/vsphere_volume/vsphere_volume.go

```diff
@@ -120,6 +120,18 @@ func (plugin *vsphereVolumePlugin) newUnmounterInternal(volName string, podUID t
 		}}, nil
 }
 
+func (plugin *vsphereVolumePlugin) ConstructVolumeSpecFromName(volumeName string) (*volume.Spec, error) {
+	vsphereVolume := &v1.Volume{
+		Name: volumeName,
+		VolumeSource: v1.VolumeSource{
+			VsphereVolume: &v1.VsphereVirtualDiskVolumeSource{
+				VolumePath: volumeName,
```

Comment:
OK. Thank for the response. I tested the patch in AWS and the purpose of the PR was mainly to solicit feedback on the patch since it's actually my first actual contribution to kubernetes.